modules/govuk_postgresql/templates/etc/cron.daily/autopostgresqlbackup.erb

```diff
@@ -344,6 +344,16 @@ dbdump () {
   touch $2
   chmod 600 $2
   for db in $1 ; do
+    <% if @environment != 'integration' %>
```

Comment:
If I'm reading this right it looks like this will exclude the table in all environments except Integration - is this correct?